Documentation/Books/Cookbook/Administration/Replication/README.md

```diff
@@ -0,0 +1,6 @@
+# Replication
+
+This Section includes cookbooks related to the *Replication* topic.
+
```

Comment:
Should be "cookbook recipes" IMO. Cookbook is everything in that Gitbook, isn't it?

```md
This section includes cookbook recipes related to the *Replication* topic.
```